src/slic3r/GUI/Field.cpp

```diff
@@ -525,28 +525,17 @@ void release_window_pools()
 template<typename T>
 struct Builder
 {
-    Builder()
+    template<typename... Args>
```

Comment:
Do you mind to explain the changes a bit?
Like what is it trying to fix?

Reply:
This is the bit that fixes #5028.  Without, most of the settings fields (TextCtrls, Checkboxes, etc) become unresponsive.

Reply:
Hey @jamincollins 
After some evaluation, I have reverted the "control pool" changes which caused these issues.
https://github.com/SoftFever/OrcaSlicer/commit/d72c9f923a12b60573674fa87b47a228b7cffb9f

Reply:
I'll test the latest changes.